### DISCUSSION TITLE
Config flex?

### DISCUSSION THREAD

**Discussion by iamkarlson** [WillPower3309/swayfx#126] [Ideas]
Some place to share configs specif to swayfx would be great so you as a new user can start using new functionality easier

**Comment by ErikReider** [comment]
We can always open a discussion for config flex. What do you think @WillPower3309 ?

**Comment by WillPower3309** [comment]
Sure! I think this could be a good idea. Potentially with screenshots as well!

**Comment by WillPower3309** [comment]
https://github.com/WillPower3309/swayfx/discussions/138
